src/main/java/org/folio/service/organization/OrganizationStorageService.java

```diff
@@ -123,15 +133,18 @@ public CompletableFuture<OrganizationCollection> getOrganizationCollection(int o
   @Override
   public CompletableFuture<Void> updateOrganizationById(String id, Organization updatedOrganization, String lang, Context context,
       Map<String, String> headers) {
+    logger.debug("updateOrganization:: Trying to update organization with id: {}", id);
     CompletableFuture<Void> future = new CompletableFuture<>();
     if (isEmpty(updatedOrganization.getId())) {
       updatedOrganization.setId(id);
     } else if (!id.equals(updatedOrganization.getId())) {
+      logger.warn("updateOrganization:: Mismatch between id in path and request body");
       future.completeExceptionally(new HttpException(HttpStatus.HTTP_UNPROCESSABLE_ENTITY.toInt(),
         MISMATCH_BETWEEN_ID_IN_PATH_AND_BODY.toError()));
       return future;
     }
     if (isSameAccountNumbers(updatedOrganization)) {
+      logger.warn("updateOrganization:: Account number is not unique");
```

Comment:
please add accountNumber to this log

Reply:
if its not possbile to add acount number - add orginazation id here

Reply:
yeah, thanks added